taglib/toolkit/tutils.h

```diff
@@ -203,11 +203,12 @@ namespace TagLib
       /*!
        * Returns the byte order of the system.
        */
-      inline ByteOrder systemByteOrder()
+      inline constexpr ByteOrder systemByteOrder()
```

Comment:
I wonder if inline is needed here...

Reply:
oh no it's not. constexpr implies inline.

Reply:
Removed it.